File: src/main/java/io/github/mg138/tsbook/command/admin/CommandUtil.java
package io.github.mg138.tsbook.command.admin;

import org.bukkit.Server;
import org.bukkit.entity.Player;

import java.util.ArrayList;
import java.util.List;

public class CommandUtil {
    public static List<String> getPlayerNames(Server server) {
        List<String> playerNames = new ArrayList<>();
        for (Player player : server.getOnlinePlayers()) {
            playerNames.add(player.getName());
        }
        return playerNames;
    }
}

File: src/main/kotlin/io/github/mg138/tsbook/command/util/CommandUtil.kt
package io.github.mg138.tsbook.command.util

import io.github.mg138.tsbook.config.BookConfig
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player
import java.util.*

object CommandUtil {
    fun sendGetMessage(receiver: CommandSender, sender: CommandSender, itemName: String) {
        val message = BookConfig.language.messages.get(receiver.name, itemName)
        sender.sendMessage(message)
        receiver.sendMessage(message)
    }

    fun getPlayerNames(players: Collection<Player>): List<String> {
        val playerNames: MutableList<String> = LinkedList()

        players.forEach {
            playerNames += it.name
        }

        return playerNames
    }
}